transmart-core-db/grails-app/services/org/transmartproject/db/clinical/PatientSetService.groovy

```diff
@@ -716,16 +731,46 @@ class PatientSetService extends AbstractDataResourceService implements PatientSe
     }
 
     /**
-     * Clear all patient sets.
-     * This function should be called after data loading, to invalidate
-     * any persisted patient sets.
+     * Clear  query result ids cache.
+     * This function should be called after data loading.
      */
-    @Transactional
-    void clearPatientSets() {
-        log.info 'Clearing patient sets ...'
-        QtPatientSetCollection.executeUpdate('delete QtPatientSetCollection')
-        QtQueryResultInstance.executeUpdate('delete QtQueryResultInstance')
-        QtQueryInstance.executeUpdate('delete QtQueryInstance')
+    void clearPatientSetIdsCache() {
```

Comment:
Might be useful to change the documentation. https://transmart-test.thehyve.net/open-api/index.html#!/v2/get_v2_admin_system_after_data_loading_update mentions clearing patient sets, so can be confusing